rasa/core/policies/ensemble.py

```diff
@@ -449,18 +450,24 @@ def _check_if_rule_policy_used_with_rule_like_policies(
             )
 
 
-class Prediction(NamedTuple):
-    """Stores the probabilities and the priority of the prediction."""
-
-    probabilities: List[float]
-    priority: int
-
-
 class SimplePolicyEnsemble(PolicyEnsemble):
+    """Default implementation of a `Policy` ensemble."""
+
     @staticmethod
     def is_not_memo_policy(
-        policy_name: Text, max_confidence: Optional[float] = None
+        policy_name: Optional[Text], max_confidence: Optional[float] = None
     ) -> bool:
+        """Checks if the policy is the `MemoizationPolicy` or one of its subclasses.
+
+        Args:
+            policy_name: The name of the policy.
+            max_confidence: The max confidence of the policy's prediction.
+
+        Returns: `True` if it's a `MemoizationPolicy`, `False` otherwise.
+        """
+        if not policy_name:
+            return True
+
         is_memo = policy_name.endswith("_" + MemoizationPolicy.__name__)
```

Comment:
@Ghostvv That should now also include the `RulePolicy`, shouldn't it?

Reply:
hm, good catch! Probably yes, since we allow all policies together now

Reply:
added + tests for it 👍